assembl/static2/js/app/components/common/navbar.jsx

```diff
@@ -6,6 +6,7 @@ import Avatar from './avatar';
 import LanguageMenu from './languageMenu';
 import NavigationMenu from './navigationMenu';
 import Search from '../search';
+import { Routes } from '../../routes';
```

Comment:
I know we have discussed moving Routes out of the routes.jsx, and into another file, but can I ask what the issue is here?